addons/block_code/simple_nodes/simple_ending/simple_ending.gd

```diff
@@ -21,12 +21,13 @@ func game_over(result: String):
 		"LOSE":
 			text = lose_message
 		_:
-			text = ""
+			reset()
 			push_warning('Unrecognized game result "%s"' % result)
 
 
 func reset():
-	text = ""
+	# Workaround for POT generation extracting empty string.
+	text = str("")
```

Comment:
Weird, so it extracts constant strings for translation even if not tagged with `tr()`?

Reply:
It's actually a little uglier than that now that I've read through [the GDScript translation parser](https://github.com/godotengine/godot/blob/master/modules/gdscript/editor/gdscript_translation_parser_plugin.cpp). The assumption is that anything that assigns to a [`text` property](https://github.com/godotengine/godot/blob/c6c464cf9ae56e8b68620af65125dd980d0e8122/modules/gdscript/editor/gdscript_translation_parser_plugin.cpp#L254) is a `Label` or similar, which should be automatically translated. In this case, it is a subclass of a `Label`, so that's a correct assumption. However, I don't think it should try to translate an empty string.

I wouldn't have cared except that `msgmerge` errored on a duplicate empty string msgid.

Reply:
Weird. I can see the appeal of not having to litter the code base with `tr()` at some level, but it seems error-prone.